src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const listOfPeople = [...document.querySelectorAll('li')];
+
+const changedList = document.querySelector('ul');
```

Comment:
listToChange

it's not changed at this moment

Reply:
fixed